DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/data/platforms.json

```diff
@@ -1,11 +1,11 @@
 {
-    "arm64-delta-tn48m-poe-dn-r0": {
```

Comment:
can we keep arm64-delta-tn48m-poe-dn-r0 and not the marketing name

Reply:
Unfortunately, we can't use `arm64-delta-tn48m-poe-dn-r0` since `onlpdump -s` on TN48M returns `Platform Name: 88F7040/88F6820`. Other than that I don't see an issue with using either platform name or product name.